README.md

```diff
@@ -1,2 +1,90 @@
 # open-sir
-Python SIR-x model implementation
+
+Open source pandemics modelling API in python. The models implemented are in the scope of compartmental models of disease transmission, such as the widely used [Susceptible-Infected-Removed (SIR) model](http://rocs.hu-berlin.de/corona/docs/forecast/model/#classic-sir-dynamics). Recently, a new epidemic model based in SIR, SIR-X, was developed by the [Robert Koch Institut](http://rocs.hu-berlin.de/corona/docs/forecast/model/#sir-x-dynamics-outbreaks-with-temporally-increasing-interventions) and it is also implemented in open-sir.
+
+The API features model construction, parameter fitting and calculation of confidence intervals.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
```

Comment:
I think this should be focused in the End User. IMO it's better to distinguish between Developer documentation and User documentation